lib/msal-browser/src/internals.ts

```diff
@@ -8,36 +8,18 @@
  * Breaking changes to these APIs will be shipped under a minor version, instead of a major version.
  */
 
-// Cache Manager
-export { BrowserCacheManager } from "./cache/BrowserCacheManager";
-
-// Cache entities
-export { CacheRecord } from "@azure/msal-common";
+// Storage
+export { MemoryStorage } from "./cache/MemoryStorage";
```

Comment:
Why are we adding new things to internals? Goal is to remove internals entirely

Reply:
We need that for temporary broker storage in 1p.

Reply:
Then let's export this in the main exports